src/test/java/calculator/StringParserTest.java

```diff
@@ -5,60 +5,50 @@
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
 
 class StringParserTest {
 
-    private void assertParsedCorrectly(String text, String expected) {
-        assertThat(StringParser.splitNumbers(text)).containsExactly(expected.split(","));
-    }
-
     @DisplayName("콤마가 포함된 문자열을 파싱할 수 있다.")
     @ParameterizedTest
-    @CsvSource(delimiter = '|', textBlock = """
-            1,2 | 1,2
-            1,2,3 | 1,2,3
-        """)
-    void comma(final String text, final String expected) {
-        assertParsedCorrectly(text, expected);
+    @ValueSource(strings = {"1,2,3"})
+    void comma(final String text) {
+        String[] expected = {"1", "2", "3"};
+        assertThat(StringParser.splitNumbers(text)).containsExactly(expected);
     }
 
     @DisplayName("콜론이 포함된 문자열을 파싱할 수 있다.")
     @ParameterizedTest
-    @CsvSource(delimiter = '|', textBlock = """
-            1:2 | 1,2
-            1:2:3 | 1,2,3
-        """)
-    void colon(final String text, final String expected) {
-        assertParsedCorrectly(text, expected);
+    @ValueSource(strings = {"1:2:3"})
+    void colon(final String text) {
+        String[] expected = {"1", "2", "3"};
+        assertThat(StringParser.splitNumbers(text)).containsExactly(expected);
```

Comment:
containsExactly 의 경우 가변인자를 받을 수 있어서 변수로 뻬는 것보다는 바로 사용하실 수 도 있어요.

```suggestion
        assertThat(StringParser.splitNumbers(text)).containsExactly("1", "2", "3");
```